packages/app/src/specs/CreateSpecModal.cy.tsx

```diff
@@ -103,3 +103,76 @@ describe('playground', () => {
     .get(modalCloseSelector)
   })
 })
+
+describe('defaultSpecFileName', () => {
+  it('shows correct default filename for e2e testing', () => {
+    const show = ref(true)
+
+    cy.mount(() => (<div>
+      <CreateSpecModal
+        gql={{
+          currentProject: {
+            id: 'id',
+            currentTestingType: 'e2e',
+            configFile: 'cypress.config.js',
+            configFileAbsolutePath: '/path/to/cypress.config.js',
+            config: [{
+              field: 'e2e',
+              value: {
+                specPattern: 'cypress/e2e/**/*.cy.{js,jsx,ts,tsx}',
+              },
+            }, {
+              field: 'component',
+              value: {
+                specPattern: '**/*.cy.{js,jsx,ts,tsx}',
+              },
+            }],
+            specs: [],
+            fileExtensionToUse: 'js',
+            defaultSpecFileName: 'spec.cy.js',
+          },
+        }}
+        show={show.value}
+        onClose={() => show.value = false}
+      />
+    </div>))
```

Comment:
Should we use `cy.mountFragment` instead?

Reply:
Yep that'll clean things up a bit. I was also able to remove one of these test cases now that we no longer flex within the component based on testing type.